framework/include/mesh/MooseMesh.h

```diff
@@ -418,11 +418,34 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   ///@}
 
   /**
-   * Returns a map of boundaries to elements.
+   * Returns a map of boundaries to ids of elements on the boundary.
    */
   const std::unordered_map<boundary_id_type, std::unordered_set<dof_id_type>> &
   getBoundariesToElems() const;
 
+  /**
+   * Returns a map of boundaries to ids of elements on the boundary.
+   */
+  const std::unordered_map<boundary_id_type, std::unordered_set<dof_id_type>> &
+  getBoundariesToActiveLocalElemIds() const;
+
+  /**
+   * Return all ids of elements which have a side which is part of a boundary/sideset.
+   * Note that boundaries are sided.
+   * @param bid the id of the boundary/sideset of interest
```

Comment:
As this is developer documentation, I think we can be 100% explicit that this is only about sidesets (nodesets be gone!)
```suggestion
   * Return all ids of elements which have a side which is part of a sideset.
   * @param bid the id of the sideset of interest
```

Reply:
forgot to click I ll fix it later

Reply:
ok coaught it